src/main/java/com/clanki/Clanki.java

```diff
@@ -2,21 +2,30 @@
 
 import com.clanki.commands.ByeCommand;
 import com.clanki.commands.Command;
+import com.clanki.objects.Flashcard;
 import com.clanki.objects.FlashcardList;
 import com.clanki.parser.Parser;
 import com.clanki.ui.Ui;
 
+import java.util.ArrayList;
+
 public class Clanki {
-    private Ui ui;
-    private FlashcardList flashcardList;
-    private Parser parser;
+    private final Ui ui;
+    private final FlashcardList flashcardList;
+    private final Parser parser;
+    private ArrayList<Flashcard> flashcardsForToday;
+    private int currentFlashcardIndex;
```

Comment:
These variables are unused. I think it might be a good idea to just remove them, if we don't need them.